src/Person.cpp

```diff
@@ -526,7 +524,7 @@ UBYTE CLANS::GetCustomerId(SLONG Browned, SLONG Koffer, TEAKRAND *pRand) {
 
     if (possibleClanIds.empty()) {
         AT_Log_Generic("Failed to find any suitable person clan for: %d %d", Browned, Koffer);
-        TeakLibW_Exception(FNL, ExcNever);
+        return static_cast<UBYTE>(0);
```

Comment:
Is clan 0 the best choice? Do you know what clan 0 looks like? (Just curious tbh)

Reply:
Honestly, no. My game kept crashing at this line because there must be some infrequent bug when generating clans. I feel that it is better to generate clan 0 in these rare cases then crashing.

Reply:
Yeah, that code produces the most crash reports across the board. I think there is a certain combination of parameters that does not have a corresponding clan for it.

Anything is better than crashing though. I'll add an issue for future reference. Let's keep it on 0 for now